httpbin/handlers.go

```diff
@@ -255,6 +255,39 @@ func (h *HTTPBin) Status(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// Unstable - returns 500, sometimes
+func (h *HTTPBin) Unstable(w http.ResponseWriter, r *http.Request) {
+	var err error
+
+	// rng/seed
+	rng, err := parseSeed(r.URL.Query().Get("seed"))
+	if err != nil {
+		http.Error(w, "invalid seed", http.StatusBadRequest)
+		return
+	}
+
+	// failureRate
+	var failureRate float64
+	rawFailureRate := r.URL.Query().Get("failureRate")
```

Comment:
For consistency with other optional parameters (e.g. `chunk_size`), let's call this `failure_rate`.

Reply:
Ah, oversight on my part. Better?